packages/material-ui-system/src/resonsivePropType.js

```diff
@@ -0,0 +1,8 @@
+import PropTypes from 'prop-types';
+
+const resonsivePropType =
```

Comment:
```suggestion
const responsivePropType =
```

Reply:
It's still there. Could at least tell my way you dismissed this review.

Reply:
Fixing.